$NetBSD: patch-sys_autoconf_Makefile.src,v 1.2 2011/10/26 23:04:35 rhialto Exp $

Add #defines for libraries as set in pkgsrc Makefile, and runtime
link path for curses.

--- sys/autoconf/Makefile.src.orig	2011-10-08 17:52:15.000000000 +0000
+++ sys/autoconf/Makefile.src
@@ -31,7 +31,7 @@ CC = @CC@ -DAUTOCONF
 WINDRES = @WINDRES@
 
 CFLAGS = -Wall -I../include -I$(srcdir)/../include @CFLAGS@ #-fbounds-check #-fprofile-arcs -ftest-coverage
-LFLAGS = @LFLAGS@ #-fbounds-check
+LFLAGS = @LFLAGS@ -Wl,-R,${CURSESLIBDIR} #-fbounds-check
 
 # Compile with PDCurses installed in a separate directory that doesn't
 # conflict with the system curses/ncurses library
